jre_emul/Classes/com/google/j2objc/security/IosRSAKey.java

```diff
@@ -209,11 +194,9 @@ private native long createPublicSecKeyRef(byte[] bytes, int keySizeInBits) /*-[
         @throw create_JavaSecurityProviderException_initWithNSString_(msg);
       }
 
+
       // Store key in keychain.
       publicKeyQuery = getQuery();  // remove if not necessary
-      NSNumber *keySize = [NSNumber numberWithInt:keySizeInBits];
-      publicKeyQuery[(id)kSecAttrEffectiveKeySize] = keySize;
-      publicKeyQuery[(id)kSecAttrKeySizeInBits] = keySize;
```

Comment:
Key size shouldn't be specified because the key is already defined, right? (Just want to understand.)